redisvl/extensions/sessionmanager/__init__.py

```diff
@@ -0,0 +1,3 @@
+from redisvl.extensions.sessionmanager.session import SessionManager
```

Comment:
wondering if we should name the module `session_manager` for better readability. WDYT?

Reply:
I wasn't sure what style to follow, but yeah, I agree `session_manager` is better.